src/graphql/types/object/transaction.ts

```diff
@@ -60,7 +60,8 @@ const Transaction = GT.Object<WalletTransaction>({
         try {
           result = await loaders.txnMetadata.load(source.id)
         } catch (err) {
-          result = err
+          if (err instanceof Error) result = err
+          result = new Error("Unknown error")
```

Comment:
this is changing the code behavior here?

Reply:
ah yes. good catch. how does this look?

```
        try {
          result = await loaders.txnMetadata.load(source.id)
        } catch (err) {
          if (err instanceof Error) {
            result = err
          } else if (typeof err === "undefined") {
            result = undefined
          } else {
            result = new Error("Unknown error")
          }
        }
```